images/libvirt-kubevirt/Dockerfile

```diff
@@ -44,4 +44,12 @@ RUN rm -f /libvirtd.sh
 COPY libvirtd.sh /libvirtd.sh
 RUN chmod a+x /libvirtd.sh
 
+RUN echo "unix_sock_group=\"libvirt\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_ro_perms=\"0777\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "unix_sock_rw_perms=\"0770\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_ro=\"none\"" >> /etc/libvirt/libvirtd.conf
+RUN echo "auth_unix_rw=\"none\"" >> /etc/libvirt/libvirtd.conf
```

Comment:
@cynepco3hahue I think what @berrange had in mind, is to supply a client configuration. Then you don't have to provide the password and the user via commandline: https://libvirt.org/auth.html#Auth_client_config